src/main/java/org/opentripplanner/standalone/config/buildconfig/NetexConfig.java

```diff
@@ -158,6 +158,29 @@ private static NetexFeedParameters.Builder mapFilePatternParameters(
           .docDefaultValue(dft.ferryIdsNotAllowedForBicycle())
           .asStringSet(base.ferryIdsNotAllowedForBicycle())
       )
+      .addRouteToCentroidStopPlaceIds(
```

Comment:
I do not see this as a netex feature - If so, there is a lot of changes needed in the main code. This is a generic feature witch can be applied to both NeTEx and GTFS.